### PR TITLE
527 529 Governance and contributing info

### LINKED ISSUE #527: Link intro to governance documentation
## Formalizing the FDC3 Standard documentation

This issue has been raised as part of work, sponsored by FINOS, to improve and formalize the documentation of the FDC3 standard.

### How is FDC3 governed

**Proposal**
Get started | Introduction contains a section on governance ["How is FDC3 governed"](https://fdc3.finos.org/docs/fdc3-intro#how-is-fdc3-governed). When the updated governance documentation is merged (under the below PR) the introduction should summarize it and link to it for more details  
- PR #485

### LINKED ISSUE #529: Describe the process of contributing to FDC3
## Formalizing the FDC3 Standard documentation

This issue has been raised as part of work, sponsored by FINOS, to improve and formalize the documentation of the FDC3 standard.

### Contributing

**Proposal**
Most contributions to FDC3 start out as issues that are discussed at meetings, consensus (of the Standards Working Group) is established that a change should be made to the standard, it's documentation or supporting software, PR(s) are implemented and reviewed by the maintainers (and an editor in future) and merged into the next 'pre-draft' of the standard.  Contributions may also be implemented as PRs before consensus is established that a change should be made, particularly those proposed by 'discussion groups' whose proposals will often be more detailed, in which case the PR is presented to the Standards Working Group and consensus checked on merging the change, alongside review by the maintainers. 

This process is a layer below the process detailed in the proposed Governance documentation for FDC3, see [this comment](https://github.com/finos/FDC3/pull/485#issuecomment-991418738) from @scooter99boston for details. It should be described in the [Contributing section of the FDC3 readme](https://github.com/finos/FDC3/pull/485#issuecomment-991418738) and the [Introduction page of the FDC3 website](https://github.com/finos/FDC3/pull/485#issuecomment-991418738)

### PR DESCRIPTION
**Pull request by kriswest**
resolves #527, resolves #529 

Adds: 
- a contribution guide to the repository based on the [CSL contribution guide 1.0](https://github.com/finos/standards-project-blueprint/edit/master/6._Contributing.md), 
- link from governance doc to scope doc
- links from fdc3-intro to the contribution guide, governance doc and chart
- links README to contribution guide and reduces duplication
- details about Editors to the README to bring it into line with the Governance doc

### CODE REVIEW

**Review by RexJaeschke — APPROVED** [review]
Looks OK to me!

File: README.md [hunk 1]
```diff
@@ -152,8 +152,11 @@ Please note that standard participants are bound to the provisions in the [FINOS
...
 
+### FDC3 Editors
+FDC3 Editors are responsible for ensuring that the contents of the document accurately reflect the decisions that have been made by the group, and that the specification adheres to formatting and content guidelines. The appointment of an Editor is approved by the FDC3 Standards Working Group Participants.
+
 ### FDC3 Maintainers
-FDC3 maintainers are responsible for organizing activities around developing, maintaining, and updating the specification developed by the FDC3 Standard Working Group. Maintainers are also responsible for determining consensus and coordinating appeals, as well as FDC3-related planning and communications. 
+FDC3 maintainers are responsible for organizing activities around developing, maintaining, and updating the specification developed by the FDC3 Standard Working Group. Maintainers are also responsible for determining consensus and coordinating appeals, as well as FDC3-related planning and communications. The appointment of an Editor is approved by the FDC3 Standards Working Group Participants.
```
Comment (kriswest) [inline README.md:159]:
```suggestion
FDC3 maintainers are responsible for organizing activities around developing, maintaining, and updating the specification developed by the FDC3 Standard Working Group. Maintainers are also responsible for determining consensus and coordinating appeals, as well as FDC3-related planning and communications. The appointment of an Maintainer is approved by the FDC3 Standards Working Group Participants.
```

File: docs/fdc3-intro.md [hunk 2]
```diff
@@ -36,7 +36,9 @@ FDC3 was launched in October 2017 by [OpenFin](http://www.openfin.co) in collabo
 
 FDC3 is hosted within, and governed by the policies of, the [Fintech Open Source Foundation](http://finos.org/) (FINOS). FINOS is an independent nonprofit organization focused on promoting open innovation within financial services.
 
-See the [FDC3 Charter](fdc3-charter#licensing) for details of how deliverables are licensed.
+- See the [FDC3 Governance document](https://github.com/finos/FDC3/blob/master/GOVERNANCE.md) for details of how FDC3 is governed.
+- See the [FDC3 Contribution Guide](https://github.com/finos/FDC3/blob/master/CONTRIBUTING.md) for details of how to contribute to FDC3.
+- See the [FDC3 Charter](fdc3-charter#licensing) for details of how deliverables are licensed.
```
Comment (greyseer256) [inline docs/fdc3-intro.md:41]: This link 404s for me

**Review by greyseer256 — APPROVED** [review]
Other than that one link this looks good to me.

**Review by rikoe — APPROVED** [review]
Excellent.

**Pull request merged**